packages/expo-updates/ios/EXUpdates/SelectionPolicy/EXUpdatesLauncherSelectionPolicy.h

```diff
@@ -0,0 +1,15 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+#import <EXUpdates/EXUpdatesUpdate.h>
+
+NS_ASSUME_NONNULL_BEGIN
+/**
+ * Given a list of updates, implementations of this protocol should be able to choose one to launch.
+ */
+@protocol EXUpdatesLauncherSelectionPolicy
+
+- (nullable EXUpdatesUpdate *)launchableUpdateWithUpdates:(NSArray<EXUpdatesUpdate *> *)updates filters:(nullable NSDictionary *)filters;
```

Comment:
possibly clearer name:
```suggestion
- (nullable EXUpdatesUpdate *)launchableUpdateFromUpdates:(NSArray<EXUpdatesUpdate *> *)updates filters:(nullable NSDictionary *)filters;
```